tests/foreman/cli/test_host.py

```diff
@@ -2683,3 +2683,71 @@ def test_positive_create_host_with_lifecycle_environment_name(
     hosts = module_target_sat.cli.Host.list({'organization-id': module_org.id})
     found_host = any(new_host.name in i.values() for i in hosts)
     assert found_host, 'Assertion failed: host not found'
+
+
+@pytest.mark.no_containers
+@pytest.mark.rhel_ver_match('8')
+@pytest.mark.parametrize(
+    'setting_update', ['validate_host_lce_content_source_coherence'], indirect=False
+)
+def test_positive_host_registration_with_capsule(
```

Comment:
The test name looks very generic, can we keep name something related to the used setting or what it does?

Reply:
> The test name looks very generic, can we keep name something related to the used setting or what it does?

It would be long if we consider the scenario or try to relate it with settings name.
Can you suggest a good one? @Gauravtalreja1

Reply:
`test_positive_host_registration_with_capsule_using_validate_content_coherence()` @Gauravtalreja1

Reply:
`test_host_registration_with_capsule_using_content_coherence()`